airbyte-integrations/connectors/source-shopify/integration_tests/expected_records.jsonl

```diff
@@ -24,9 +24,9 @@
-{"stream": "inventory_levels", "data": {"id": "63590301885|42185218719933", "available": 8, "updated_at": "2023-04-13T12:09:45+00:00", "admin_graphql_api_id": "gid://shopify/InventoryLevel/97912455357?inventory_item_id=42185218719933", "inventory_item_id": 42185218719933, "location_id": 63590301885, "shop_url": "airbyte-integration-test"}, "emitted_at": 1708953807904}
-{"stream": "inventory_levels", "data": {"id": "63590301885|43653688524989", "available": 0, "updated_at": "2023-04-24T18:00:10+00:00", "admin_graphql_api_id": "gid://shopify/InventoryLevel/97912455357?inventory_item_id=43653688524989", "inventory_item_id": 43653688524989, "location_id": 63590301885, "shop_url": "airbyte-integration-test"}, "emitted_at": 1708953807905}
+{"stream": "inventory_levels", "data": {"id": "63590301885|42185200631997", "updated_at": "2023-04-13T12:00:55+00:00", "quantities": [{"id": 97912455357, "name": "available", "quantity": 15, "updatedAt": "2023-04-13T12:00:55+00:00", "admin_graphql_api_id": "gid://shopify/InventoryQuantity/97912455357?inventory_item_id=42185200631997&name=available"}, {"id": 97912455357, "name": "incoming", "quantity": 0, "updatedAt": null, "admin_graphql_api_id": "gid://shopify/InventoryQuantity/97912455357?inventory_item_id=42185200631997&name=incoming"}, {"id": 97912455357, "name": "committed", "quantity": 0, "updatedAt": "2022-03-17T10:05:08+00:00", "admin_graphql_api_id": "gid://shopify/InventoryQuantity/97912455357?inventory_item_id=42185200631997&name=committed"}, {"id": 97912455357, "name": "damaged", "quantity": 0, "updatedAt": null, "admin_graphql_api_id": "gid://shopify/InventoryQuantity/97912455357?inventory_item_id=42185200631997&name=damaged"}, {"id": 97912455357, "name": "on_hand", "quantity": 15, "updatedAt": "2023-04-13T12:00:52+00:00", "admin_graphql_api_id": "gid://shopify/InventoryQuantity/97912455357?inventory_item_id=42185200631997&name=on_hand"}, {"id": 97912455357, "name": "quality_control", "quantity": 0, "updatedAt": null, "admin_graphql_api_id": "gid://shopify/InventoryQuantity/97912455357?inventory_item_id=42185200631997&name=quality_control"}, {"id": 97912455357, "name": "reserved", "quantity": 0, "updatedAt": null, "admin_graphql_api_id": "gid://shopify/InventoryQuantity/97912455357?inventory_item_id=42185200631997&name=reserved"}, {"id": 97912455357, "name": "safety_stock", "quantity": 0, "updatedAt": null, "admin_graphql_api_id": "gid://shopify/InventoryQuantity/97912455357?inventory_item_id=42185200631997&name=safety_stock"}], "admin_graphql_api_id": "gid://shopify/InventoryLevel/97912455357?inventory_item_id=42185200631997", "inventory_item_id": 42185200631997, "location_id": 63590301885, "shop_url": "airbyte-integration-test"}, "emitted_at": 1716464698314}
```

Comment:
In [the issue description](https://github.com/airbytehq/airbyte-internal-issues/issues/7357), I see you mentioning that `inventory_levels stream will deprecate the availability`. Do we mean `available` or `availability`? I just want to make sure as I don't have a lot of experience for Shopify

Reply:
Typo. Fixed the description.